.flake8

```diff
@@ -12,7 +12,8 @@ ignore = E203, E501, W503,  # handled by black
          S311,
          T001,
          B007
-         S608
+         S608,
+         PT012
```

Comment:
"PT012 pytest.raises() block should contain a single simple statement" needs to help ensure that only the expected exception from code under test is caught in a pytest.raises block (e.g. not an exception from initialization/finalization code)

As I undersrand, it wants us to write like this:
`    with suppressor_generator(IndexError):`
`        with pytest.raises(IndexError):`
`                  [][2]`

and there would be no "PT012". But, here we can test without `with pytest.raises(IndexError):` . So, just for another one check that out  suppressor_generator works correct, we need to write like this:
`    with pytest.raises(IndexError):`
`         with suppressor_generator(IndexError):`
`                  [][2]`
Here, [][0] we had IndexError that suppressed by suppressor_generator(IndexError), and then **test will fail** because `with pytest.raises(IndexError):`  didn`t raised IndexError, because it was already suppressed by suppressor_generator(IndexError).